### PR TITLE
Lps 183203 

### PR DESCRIPTION
**Pull request by yanan**
Background
- Add tests for Story LPS-177291 hide options in import/export for entities

Test Plan
- https://issues.liferay.com/browse/LPS-183179

Jira Ticket
- https://issues.liferay.com/browse/LPS-183203

### CODE REVIEW

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:sf

**Comment by yanan** [comment]
ci:test:sf

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Review by magjed4289 — APPROVED** [review]
just one dount, the rest looks well!

**Review by Hijky — APPROVED** [review]

File: modules/apps/batch-planner/batch-planner-test/src/testFunctional/macros/ImportExport.macro [hunk 1]
```diff
@@ -181,6 +181,31 @@ definition {
...
+
+		for (var entityType : list ${entityTypes}) {
+			ImportExport.selectEntity(entityType = ${entityType});
+
+			if (isSet(importStrategy)) {
+				AssertTextEquals(
+					locator1 = "ImportExport#IMPORT_STRATEGY",
+					value1 = ${importStrategy});
+
+				//ImportExport.assertAvailableUpdateStrategies(updateStrategies = ${updateStrategies});
```
[the remark this answers is not in the document]
Comment (yanan) [reply modules/apps/batch-planner/batch-planner-test/src/testFunctional/macros/ImportExport.macro]: I already removed that which being used during testing, thanks so much!

**Comment by yanan** [comment]
ci:test:sf

**Comment by yanan** [comment]
ci:forward

**Pull request closed without merging**